src/components/v2/Delimiter/index.tsx

```diff
@@ -0,0 +1,14 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+
+import { useDelimiterStyles as useStyles } from './styles';
+
+interface IDelimiterProps {
+  className?: string;
+  spaceSize?: 3 | 4 | 5;
```

Comment:
I'd remove this prop. We should extend the delimiter's styling if we want to apply margins to it from a parent component, this way we're consistent with the rest of the codebase.

Reply:
done